.github/workflows/release.yaml

```diff
@@ -100,6 +111,36 @@ jobs:
           AC_CERTIFICATE_PASSWORD: ${{ secrets.AC_CERTIFICATE_PASSWORD }}
           AC_APIKEY_P8_BASE64: ${{ secrets.AC_APIKEY_P8_BASE64 }}
 
+      - name: Create release tag and release notes
+        run: |
+          ref=HEAD
+          old_version="$(git describe --abbrev=0 "$ref^1")"
```

Comment:
Review: Technically we should look at the latest published release on GH here, in theory one release could fail before being fully published. If we then release another we'd want the changelog to include everything.

Example: v0.14.2 (v0.14.1...v0.14.2) fails, we add a commit or two and do a new release v0.14.3. Now the changelog would only include the commits from v0.14.2...v0.14.3 even though we might want it to be v0.14.1...v0.14.3. I think it's OK to ignore for now but this leads to a state where `release.sh` tells you one thing and CI does another.

Reply:
If v0.14.2 fails the tag should be undone, either automatically by the script or manually by whoever is running the release.

Reply:
Tags are generally considered immutable, replacing them can lead conflicts in developers/contributors local repos.

Reply:
I can delay pushing the tag in CI until right before creating the release. But there's still a chance that the actual `gh release` command fails.